src/reducers/featureFlagsReducer.js

```diff
@@ -36,7 +38,14 @@ export type FeatureFlagsReducerAction = {|
 |};
 
 export const initialState = {
-  data: INITIAL_FEATURE_FLAGS,
+  data: {
+    offersEngine: true,
+    ramp: true,
+    wyre: true,
+    aave: false,
+    poolTogether: false,
```

Comment:
Sorry, i have just noticed these defaults too. Once this is changed.- we should be good to go.